definitions/pipelines/bisulfite.cwl

```diff
@@ -30,6 +30,12 @@ inputs:
         type: int
     QCannotation:
         type: File
+#nonmethylation_sites_bed is a variable to determine if user wants to obtain vcf/bed/bedgraph/bigwig for non - methylation sites. Value has to be yes or no
+    assay_non_cpg_sites:
+        type:
+            type: enum
+            symbols: ["yes", "no"]
+        default: "no"
```

Comment:
Sorry to be all nit-picky, I think I prefer "true/false", just to stay as consistent as possible across tools/params.

Reply:
Changed to true/false.